service/README.md

```diff
@@ -0,0 +1,33 @@
+# Service lifecycle
+
+Service, being a bit of an overloaded term, requires some
+specific care to explain the various components in the `service`
+package in UberFx.
+
+## Instantiation
+
+Generally, you create a service in one of two ways:
+
+* The builder pattern, e.g. `service.WithModules(...).Build()`
+* Calling `service.New()` directly.
+
+The former is generally much easier, and used in all the examples, but `New` is exported
+in case you'd like extra control over how your service is instantiated.
+
+If you **do** call `service.New()`, you will need to call `AddModules(...)` to configure
+which modules you'd like to serve.
+
+## Options
+
+Both the builder pattern and the `New()` functions take a variadic `Options`
```

Comment:
Should "functions" be "function," or does the builder pattern count as a function as well?